README.md

```diff
@@ -29,6 +29,26 @@ Or install it yourself as:
 
 TODO: Write usage instructions here
 
+#### ManageIQ::Api::Common::Filter
```

Comment:
We can probably remove the TODO on line 30 since you have instructions here for one part of the gem.

Reply:
done!